modules/apps/layout/layout-type-controller/layout-type-controller-asset-display/src/main/java/com/liferay/layout/type/controller/asset/display/internal/portlet/AssetDisplayPageFriendlyURLProviderImpl.java

```diff
@@ -160,6 +165,29 @@ private String _getFriendlyURL(
 				layoutDisplayPageObjectProvider.getClassPK(),
 				layoutDisplayPageObjectProvider.getClassTypeId())) {
 
+			AssetRendererFactory<?> assetRendererFactory =
+				AssetRendererFactoryRegistryUtil.
+					getAssetRendererFactoryByClassName(
+						layoutDisplayPageObjectProvider.getClassName());
+
+			AssetRenderer<?> assetRenderer =
+				assetRendererFactory.getAssetRenderer(
+					layoutDisplayPageObjectProvider.getClassPK());
+
+			try {
+				String friendlyURL = assetRenderer.getURLViewInContext(
+					themeDisplay, StringPool.BLANK);
+
+				if (!friendlyURL.equals(StringPool.BLANK)) {
```

Comment:
@bakayattila, better use `!Validator.isBlank(friendlyURL)` here. You can send a follow up, I'm forwarding :)

Reply:
@jkappler, Done as it was stable was failing I did the modification quickly.